thirdparty/runtime_environments_test.go

```diff
@@ -21,3 +22,17 @@ func TestGetImageNames(t *testing.T) {
 	assert.NotEmpty(result)
 	assert.NotContains(result, "")
 }
+
+func TestGetOSInfo(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetOSInfo")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	result, err := c.getOSInfo(ctx, "ami-0e12ef25a5f7712a4", 0, 10)
+	require.NoError(t, err)
+	assert.NotEmpty(result)
```

Comment:
you dont have to check for NotEmpty and also length on the next line 
not empty is just length != 0 so checking that the length is 10 would be enough testing